README.md

```diff
@@ -109,6 +109,13 @@ So, you can run:
 
 This approach might be the least straightforward to set up but is the most flexible as you can freely and directly interact with the runtime of the application.
 
+#### HTTPS details
+In order for Cookies to work correctly, we needed to use HTTPS in development, so that we could use local servers to test frontend changes in netlify.
+
+If you use Docker, the certs will be automatically generated and used, if you are running npm directly, you should run `./certs/certgen.sh` so that it generates the required files.
```

Comment:
```suggestion
If you use Docker, the certs will be automatically generated and used. If you are running npm directly, you should run `./certs/certgen.sh` so that it generates the required files.
```